solr/core/src/test/org/apache/solr/request/macro/TestMacroExpander.java

```diff
@@ -37,8 +37,7 @@ public void testExamples() {
       me = new MacroExpander(testParams, failOnMissingParams);
     }
 
-    // default examples: https://cwiki.apache.org/confluence/display/solr/Parameter+Substitution
-    // and http://yonik.com/solr-query-parameter-substitution/
+    // default examples: https://yonik.com/solr-query-parameter-substitution/
```

Comment:
While this is fine, it does remind me that we should probably have a community discussion about how do we get more blogs that ahve great content integrated (@gerlowskija was pushing on that).   And maybe we start going through content and bringing it into the ref guide....